statsmodels/genmod/generalized_estimating_equations.py

```diff
@@ -1122,26 +1127,42 @@ def fit(self, maxiter=60, ctol=1e-6, start_params=None,
                           ConvergenceWarning)
             return None
 
-        bcov, ncov, bc_cov, _ = self._covmat()
+        bcov, ncov, _ = self._covmat()
         if bcov is None:
             warnings.warn("Estimated covariance structure for GEE "
                           "estimates is singular", ConvergenceWarning)
             return None
+        bc_cov = None
+        if cov_type == "bias_reduced":
+            bc_cov = self._bc_covmat(ncov)
 
         if self.constraint is not None:
-            mean_params, bcov = self._handle_constraint(mean_params, bcov)
+            if cov_type == "robust":
+                cov = bcov
+            elif cov_type == "naive":
+                cov = ncov
+            elif cov_type == "bias_reduced":
+                cov = bc_cov
+
+            mean_params, cov = self._handle_constraint(mean_params, cov)
             if mean_params is None:
                 warnings.warn("Unable to estimate constrained GEE "
                               "parameters.", ConvergenceWarning)
                 return None
+            if cov_type == "robust":
+                bcov = cov
+            elif cov_type == "naive":
+                ncov = cov
+            elif cov_type == "bias_reduced":
+                bc_cov = cov
```

Comment:
given that bcov and ncov are also attached to the results instance, I think they also need to be transformed to the unconstrained parameter space, don't they?

`bcov = self.constraint.unpack_cov(bcov)`

I'll add a general question in main comment